server/src/main/scala/drt/server/feeds/acl/AclFeed.scala

```diff
@@ -242,9 +241,51 @@ object AclFeed {
   }
 
   def aclToPortMapping(portCode: PortCode): Terminal => Terminal = portCode match {
```

Comment:
May be good to have unit test for it ?  Couple of places we are multiple T=>T are return and I am not sure if .getorElse will get executed here . May it is alright just test will help to clarify my doubt.

Reply:
The `getOrElse` on maps are fine
Tests wise, it's hard to say if they'd be worth it as this is essentially just config

Reply:
can't hurt to add a test! I'll stick one in.